### PR TITLE
Improve Logging

### PR DESCRIPTION
**Pull request by TobiCode**

### CODE REVIEW

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java [hunk 1]
```diff
@@ -97,8 +97,13 @@ public void create(String appName, ApplicationBean bean, boolean shouldStart) th
         }
 
         try {
+            Log.debug("Create app:", appName,
+                    "\n bean of the app:", bean,
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java:101]: Please avoid newline characters in log messages; one line = one log message. Should be logged as three messages, I guess.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java [hunk 2]
```diff
@@ -97,8 +97,13 @@ public void create(String appName, ApplicationBean bean, boolean shouldStart) th
...
+                    "\n should start:", shouldStart);
+
             doCreate(appName, bean, shouldStart);
         } catch (RuntimeException e) {
+            Log.debug("Clean up the app you tried to create.");
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java:106]: No need to terminate log messages in any kind of punctuation.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java [hunk 3]
```diff
@@ -44,6 +44,9 @@ public ServicesOperations(DefaultCloudFoundryOperations cloudFoundryOperations)
      * @throws CreationException when the creation or the binding was not successful
      */
     public void create(String serviceInstanceName, ServiceBean serviceBean) throws CreationException {
+        Log.debug("Create service:", serviceInstanceName,
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java:47]: Please split up into 2 messages.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java [hunk 4]
```diff
@@ -70,6 +72,9 @@ public void create(String serviceInstanceName, ServiceBean serviceBean) throws C
      * @throws CreationException when the creation or the binding was not successful
      */
     public void renameServiceInstance(String newName, String currentName) throws CreationException {
+        Log.debug("Rename service:", currentName,
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java:75]: Please split up into 2 messages.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java [hunk 5]
```diff
@@ -93,6 +97,8 @@ public void renameServiceInstance(String newName, String currentName) throws Cre
      * @throws CreationException when the creation or the binding was not successful
      */
     public void updateServiceInstance(String serviceInstanceName, ServiceBean serviceBean) throws CreationException {
+        Log.debug("Update service Instance:", serviceInstanceName,
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java:100]: Please split up into 2 messages.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java [hunk 6]
```diff
@@ -66,6 +66,8 @@ public SpaceDevelopersOperations(DefaultCloudFoundryOperations cfOperations) {
      * @throws IllegalArgumentException when username is empty
      */
     public void assignSpaceDeveloper(String username) throws CreationException {
+        Log.debug("Assign a space developer:", username);
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java:69]: No need for an imperative style. I'd suggest "Assigning space developer %s".

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java [hunk 7]
```diff
@@ -104,6 +105,8 @@ public void assignSpaceDeveloper(String username) throws CreationException {
      * @throws InvalidOperationException if usernameList is null/empty or spaceId is invalid.
      */
     public void removeSpaceDeveloper(List<String> usernameList) throws InvalidOperationException {
+        Log.debug("Remove space developer(s):", String.valueOf(usernameList));
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java:108]: "Removing"

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java [hunk 8]
```diff
@@ -49,6 +49,7 @@ public Integer call() {
 
         @Override
         public Integer call() throws Exception {
+            Log.info("Assign space-developer(s) running...");
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java:52]: "Assigning"

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java [hunk 9]
```diff
@@ -74,6 +78,7 @@ public Integer call() throws Exception {
 
         @Override
         public Integer call() throws Exception {
+            Log.info("Create service(s) running...");
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java:81]: "Creating service(s)..." The three dots imply that there will be a "reply" soon, e.g., "Created services". This is the case here.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java [hunk 10]
```diff
@@ -44,13 +45,16 @@ public Integer call() throws Exception {
 
         @Override
         public Integer call() throws Exception {
+            Log.info("Remove space developers..." );
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java:48]: "Removing"

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java [hunk 11]
```diff
@@ -73,13 +77,17 @@ public Integer call() throws Exception {
             DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
             ServicesOperations servicesOperations = new ServicesOperations(cfOperations);
 
+            Log.info("Update services..." );
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java:80]: "Updating"

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java [hunk 12]
```diff
@@ -73,13 +77,17 @@ public Integer call() throws Exception {
...
             for (Entry<String, ServiceBean> serviceEntry : serviceBeans.entrySet()) {
                 String serviceName = serviceEntry.getKey();
                 ServiceBean serviceBean = serviceEntry.getValue();
 
                 // "currentName" is currently a placeholder until diff is implemented
                 servicesOperations.renameServiceInstance(serviceName, "currentName");
+                Log.info("Service name changed: " , serviceName);
                 servicesOperations.updateServiceInstance(serviceName, serviceBean);
+                Log.info("Service Plan and Tags haven been updated of service:", serviceName);
+
```
Comment (M4eqP) [inline cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java:90]: Unneeded blank line.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java (hunk 1 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java:101]: Done.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java (hunk 2 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java:106]: Removed the dot

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java (hunk 3 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java:47]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java (hunk 4 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java:75]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java (hunk 5 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java:100]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java (hunk 8 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java:52]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java (hunk 9 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java:81]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java (hunk 10 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java:48]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java (hunk 11 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java:80]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java (hunk 12 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java:90]: Removed blank line.

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java (hunk 6 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java]: Done

File: cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java (hunk 7 above)
Comment (TobiCode) [reply cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/SpaceDevelopersOperations.java]: Done

**Pull request merged**